src/worker.py

```
@@ -1134,6 +1133,12 @@ async def serve_static(path: str, env):
 # ---------------------------------------------------------------------------
 
 async def _dispatch(request, env):
+    global _CORS_ORIGIN_INITIALISED
```

Ruff (0.15.7)

[warning] Using the global statement to update `_CORS_ORIGIN_INITIALISED` is discouraged

(PLW0603)